### ISSUE TITLE
[Popover] option to render lazily

### ISSUE THREAD

**Issue by alexciesielski** [ng-matero/extensions#116]
Similarly to how tabs are able to be instantiated lazily, meaning the tab's content gets inserted into the DOM only after the tab has been activated, it would be nice to have the same ability with the popover.

https://material.angular.io/components/tabs/examples#tab-group-lazy-loaded

By passing an `ng-template` a tab's content can be rendered lazily which saves resources.

```ts
<mat-tab-group>
  <mat-tab label="First">
    <ng-template matTabContent>
      Content 1 - Loaded: {{getTimeLoaded(1) | date:'medium'}}
    </ng-template>
  </mat-tab>
  <mat-tab label="Second">
    <ng-template matTabContent>
      Content 2 - Loaded: {{getTimeLoaded(2) | date:'medium'}}
    </ng-template>
  </mat-tab>
  <mat-tab label="Third">
    <ng-template matTabContent>
      Content 3 - Loaded: {{getTimeLoaded(3) | date:'medium'}}
    </ng-template>
  </mat-tab>
</mat-tab-group>
```

**Comment by nzbin** [comment]
v14.3.0 has supported lazy rendering, please have a try.
